public/content/en/basics/delegates.md

```diff
@@ -41,16 +45,25 @@ standard function
 [`std.functional.toDelegate`](https://dlang.org/phobos/std_functional.html#.toDelegate)
 converts a `function` to a `delegate`.
 
-Nameless function which are called *lambdas* can be defined in two ways:
+### Anonymous functions & Lambdas
+
+As functions can be saved as variables and passed to other functions,
+it is laborious to give them an own name and to define them. Hence D allows
+nameless functions and one-line _lambdas_.
 
     auto f = (int lhs, int rhs) {
         return lhs + rhs;
     };
-    auto f = (int lhs, int rhs) => lhs + rhs;
+    auto f = (int lhs, int rhs) => lhs + rhs; // Lambda - internally converted to the above
+
+It is also possible to pass-in strings as template argument to functional parts
```

Comment:
Let's move the triple classes, interfaces and templates before delegated them we should be fine. It fits wuite good here sp no other idea unfortunately. I agree that it is a common pattern worth mentioning.